### DISCUSSION TITLE
关于拖拽编辑的按钮问题

### DISCUSSION THREAD

**Discussion by cyt68** [alibaba/designable#113] [Q&A] [answered]
请问拖拽表单如果不通过手写代码，能实现确定或者提交按钮吗？现在没有按钮组件，表单上也没有类似submit的配置，如果只能通过代码来加感觉拖拽有点不完整。。

**Comment by janryWang** [comment, marked as answer]
又没有不允许拖按钮，需要，就自己扩展

**Reply by cyt68** [reply]
请问是加一个按钮组件好一点，还是在表单上加一个拓展配置项如submit来自动加按钮好一点呢？另外有一个问题就是编辑器的最外层，表单这一层为啥不能配置数据源和响应器规则啊，比如我想在最外层调一个接口，然后把获取到的数据分配给表单里的组件，要如何实现呢？
感谢

**Reply by janryWang** [reply]
不要问为什么不能，啥都能，就看你咋定制
